### PR TITLE
User dashboard

### PR DESCRIPTION
**Pull request by heron2014**
Related to [issue 215](https://github.com/FAC-GM/app/issues/215)

Current wireframe [here](https://github.com/FAC-GM/app/issues/215#issuecomment-179163278)

@nelsonic please have a look at this PR, thank you

### CODE REVIEW

File: views/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    {{#each placement}}
+      <div class="contact">
+         <a href="/candidate/{{id}}"><img src="{{picture}}" alt="{{fullname}}"></a>
+        <p><a href="/candidate/{{id}}">{{fullname}}</a></p>
+      </div>
+    {{/each}}
+    </div>
+  </div>
+</div>
+
```
Comment (SimonLab) [inline views/dashboard.html]: Empty lines below to be deleted

**Comment by nelsonic** [comment]
@SimonLab are you still tidying up?

**Comment by iteles** [comment]
@nelsonic Simon is done!

**Comment by nelsonic** [comment]
Nice one. :+1:

**Pull request merged**